core/src/main/python/wlsdeploy/tool/deploy/odl_deployer.py

```diff
@@ -0,0 +1,304 @@
+"""
+Copyright (c) 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+"""
+from java.io import File
+from java.io import FileInputStream
+from java.io import FileOutputStream
+from java.io import IOException
+from java.lang import IllegalArgumentException
+from javax.xml.parsers import ParserConfigurationException
+
+from oracle.weblogic.deploy.util import FileUtils
+from org.xml.sax import SAXException
+
+from wlsdeploy.aliases import alias_utils
+from wlsdeploy.aliases.location_context import LocationContext
+from wlsdeploy.aliases.model_constants import ODL_CONFIGURATION
+from wlsdeploy.aliases.model_constants import MODEL_LIST_DELIMITER
+from wlsdeploy.aliases.wlst_modes import WlstModes
+from wlsdeploy.exception.expection_types import ExceptionType
+from wlsdeploy.logging.platform_logger import PlatformLogger
+from wlsdeploy.tool.util.alias_helper import AliasHelper
+from wlsdeploy.util import dictionary_utils
+
+from oracle.core.ojdl.weblogic.ODLConfiguration import CONFIG_DIR
```

Comment:
When running a regular createDomain in a non-jrf, this fails trying to load the ODL.
```
Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

Traceback (most recent call last):
  File "C:\Users\crountre\temp\testfiles-12.2.1.4\weblogic-deploy\lib\python\create.py", line 38, in <module>
    from wlsdeploy.tool.create.domain_creator import DomainCreator
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\create\domain_creator.py", line 84, in <module>
    from wlsdeploy.tool.deploy import model_deployer
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\deploy\model_deployer.py", line 12, in <module>
    from wlsdeploy.tool.deploy.resources_deployer import ResourcesDeployer
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\deploy\resources_deployer.py", line 10, in <module>
    from wlsdeploy.tool.deploy.odl_deployer import OdlDeployer
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\deploy\odl_deployer.py", line 25, in <module>
    from oracle.core.ojdl.weblogic.ODLConfiguration import CONFIG_DIR
ImportError: No module named core
Problem invoking WLST - Traceback (most recent call last):
  File "C:\Users\crountre\temp\testfiles-12.2.1.4\weblogic-deploy\lib\python\create.py", line 38, in <module>
    from wlsdeploy.tool.create.domain_creator import DomainCreator
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\create\domain_creator.py", line 84, in <module>
    from wlsdeploy.tool.deploy import model_deployer
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\deploy\model_deployer.py", line 12, in <module>
    from wlsdeploy.tool.deploy.resources_deployer import ResourcesDeployer
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\deploy\resources_deployer.py", line 10, in <module>
    from wlsdeploy.tool.deploy.odl_deployer import OdlDeployer
  File "C:\Users\crountre\temp\TESTFI~1.4\WEBLOG~1\lib\python\wlsdeploy\tool\deploy\odl_deployer.py", line 25, in <module>
    from oracle.core.ojdl.weblogic.ODLConfiguration import CONFIG_DIR
ImportError: No module named core
```

Reply:
Made these imports local to update method, after the JRF check has been done.